tests/bootstrap.php

```diff
@@ -1,3 +1,7 @@
 <?php
 
 require_once __DIR__ . '/../vendor/autoload.php';
+
+if (class_exists('\PHPUnit_Framework_Error_Notice')) {
+    class_alias('\PHPUnit_Framework_Error_Notice', '\PHPUnit\Framework\Error\Notice');
```

Comment:
really required? we dont use it directly. 

but doesnt matter

Reply:
It really required ad it'used [here](https://github.com/adhocore/php-underscore/blob/master/tests/UnderscoreCollectionTest.php#L38).